### PR TITLE
Conversion method for final implicit class cannot be overridden

### PR DESCRIPTION
**Pull request by Kobenko**
https://github.com/scala/bug/issues/11078

### CODE REVIEW

**Comment by Kobenko** [comment]
@retronym @adriaanm may I ask you to review?

File: test/files/neg/override-final-implicit.scala [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package files.neg
```
Comment (som-snytt) [inline test/files/neg/override-final-implicit.scala]: BTW, not necessary.

**Review by som-snytt — APPROVED** [review]
I also checked that privateWithin is preserved. The only test with a private implicit class is a test for `-Ywarn-unused`. It might be nice to have tests for access (in the same test file), but not a big deal. If it ain't broke don't test it? I see it's a similar fix on dotty, two bugs with one stone almost.

**Comment by som-snytt** [comment]
BTW, bug number not necessary or desirable in commit message, see the history on the issue with your push references.

File: test/files/neg/override-final-implicit.scala (hunk 1 above)
Comment (Kobenko) [reply test/files/neg/override-final-implicit.scala]: OK. I understood you. I will add a test and remove the package (although I will honestly say that using the package was peeped in other tests :)). and make a force-push with the new commit name

Comment (som-snytt) [reply test/files/neg/override-final-implicit.scala]: конечно понимаю. partest should wrap test code in synthetic package.

Comment (Kobenko) [reply test/files/neg/override-final-implicit.scala]: I decided that the negative test will be good in another file, ok?

**Comment by Kobenko** [comment]
@joshlemer @SethTisue can you help in order to understand who else can be to approve PR? :)

**Comment by joshlemer** [comment]
@Kobenko I think you've tagged the right people, @retronym , @adriaanm , @som-snytt and @SethTisue. I don't have any experience working on the compiler so can't really approve/comment on the PR.

**Comment by Kobenko** [comment]
very sad here.

**Comment by som-snytt** [comment]
The winter holidays can be a difficult time. Hang in there!

**Review by lrytz — APPROVED** [review]
Thank you, @Kobenko! And sorry for our delays on the PR queue..

**Pull request merged**